### DISCUSSION TITLE
Highlighted search matches

### DISCUSSION THREAD

**Discussion by joeedhorton** [UKHomeOffice/design-system#369] [Ideas]
## What
Highlighting search matches helps users identify matching data.

<img width="751" alt="Screenshot 2020-09-22 at 10 50 10" src="https://user-images.githubusercontent.com/55453490/93876051-0ecd0080-fcce-11ea-8a14-4e0ee3c43834.png">

## Why
We have removed this component from our design system for the following reasons: 
- no research has been conducted on this component in a live service
- highlighted text is not highlighted to a screenreader as important

With a lack of research and accessibility concerns, we therefore do not recommend using this component without conducting research with your users and contacting the accessibility team.

## Anything else
If you've used highlighted search matches in your service and seen them tested with users, please add your experiences in the comments below.

**Comment by isobelseacombe** [comment]
I've noticed that this isn't accessible via the menu of components in the design system - is it possible to add it to the menu?

Also, in the code snippets here https://design.homeoffice.gov.uk/components/highlighting-matches it says it is no longer in the design system. It is a component that is being tested for use in PPPT and DSA currently.

Use case in PPPT - Searching for a record with multiple search criteria. The highlight is a useful as a way of identifying which parts of the search results are matched to the search terms entered by the user

**Comment by isobelseacombe** [comment]
Can this issue be moved into a project please (I'm not sure how to do it). Where should it go - backlog or design discussion?

**Comment by karypun** [comment]
I need to document this backlog before removing it as it's tied to a few places in the current design system. Can you use the design discussion instead? There may be an issue with some of the sharing access so let me know if you run into it there.
